### ISSUE TITLE
Next steps tool is missing an accessible name

### ISSUE THREAD

**Issue by LJWatson** [w3c/transitions#34]
The `<input>``` for entering a short name doesn't have an accessible name (so it isn't identified correctly by some screen readers). Suggest using `aria-labelledby` to associate the visible "short name" text with the input.
